src/core/CCFun.mli

```diff
@@ -79,6 +66,26 @@ val iterate : int -> ('a -> 'a) -> 'a -> 'a
     [x], [iterate 1 f x] is [f x], [iterate 2 f x] is [f (f x)], etc.
     @since 2.1 *)
 
+(** {2 Infix}
+
+    Infix operators. *)
+
+module Infix : sig
+
+  val (|>) : 'a -> ('a -> 'b) -> 'b
+  (** [x |> f] is the same as [f x]. A 'pipe' operator. *)
+
+  val (@@) : ('a -> 'b) -> 'a -> 'b
+  (** [f @@ x] is the same as [f x], but right-associative.
+      @since 0.5 *)
+
+  val (%>) : ('a -> 'b) -> ('b -> 'c) -> 'a -> 'c
+  (** [(f %> g) x] or [(%>) f g x] is [g (f x)]. Alias to [compose]. *)
+
+  val (%) : ('b -> 'c) -> ('a -> 'b) -> 'a -> 'c
+  (** [(f % g) x] or [(%) f g x] is [f (g x)]. Mathematical composition. *)
+end
+
```

Comment:
I think you need to `include Infix` here, or `include module type of Infix`. A test is required, because we could have missed that and introduced a regression :)

Reply:
Oh wow, good catch!  I added an `include module type of Infix` in f7fbf8f0.  When you say "a test is required", do you mean I should add some here, or as something that should be done in a later PR/eventually?  :p

Reply:
it'd be nice to do it now, it can be as simple as this in the .ml:

```ocaml
(*$T
  CCFun.((succ % succ) 2 = 4)
  CCFun.((succ %> string_of_int) 2 = "3")
*)
```

Reply:
Is the `CCFun` necessary?  (I'm assuming this would go in `CCFun.ml` to begin with.)

Reply:
Nevermind--it appears that I don't actually know how qtest works.  Is there anyway to do this using a `$=` instead of `$T`?  I gave it a try, but it doesn't seem to like what I'm writing.

Reply:
Anyway, added a handful of $T tests in fee29724; happy to change them if needed!

Reply:
both would work (`$=` takes 2 arguments), this should do. Thanks for adding tests! :100: